pkgs/by-name/sw/swayfx-unwrapped/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  fetchFromGitHub,
+  sway-unwrapped,
+  stdenv,
+  systemd,
+  # Used by the NixOS module:
+  isNixOS ? false,
+  enableXWayland ? true,
+  systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemd,
+  trayEnabled ? systemdSupport,
+}:
+
+sway-unwrapped.overrideAttrs (
+  oldAttrs: rec {
```

Comment:
```suggestion
sway-unwrapped.overrideAttrs (oldAttrs: rec {
```

Reply:
This is how the RFC 101/166 nixfmt formats currently. 
I do agree that having it on one line looks better, tho.

Reply:
I hope that is not the intended outcome of the RFC because that looks indeed wonky